src/services/ui/src/components/RHF/TextDisplay.tsx

```diff
@@ -0,0 +1,49 @@
+import { cn } from "@/utils";
+import { Link } from "react-router-dom";
+import { RHFTextField } from "shared-types";
+
+interface RHFTextDisplayProps {
+  text: RHFTextField;
+}
+
+export const RHFTextDisplay = (props: RHFTextDisplayProps) => {
+  if (!Array.isArray(props.text)) return props.text;
+
+  return (
+    <>
+      {...props.text?.map((t) => {
+        if (typeof t === "string") return <>{t}</>;
+
+        switch (t?.type) {
+          case "br":
+            return (
+              <>
+                <br /> {t.text}
+              </>
+            );
+          case "brWrap":
+            return (
+              <>
+                <br /> {t.text} <br />
+              </>
+            );
```

Comment:
<br> is the normal way to newline from within a p tag afaik (the assumed generic case because of label wrappers, etc) but if you know of a different way to line break cleanly I'm fully open for it.